src/components/grapher/Grapher.vue

```diff
@@ -1,81 +1,92 @@
 <template>
   <div class="grapher">
-    <svg class="grapher-svg">
-      <g>
-        <!-- Note:Inside svg, 1px = 1 eight-to-five step -->
-        <g class="grapher--line-container">
-          <rect
-            class="grapher--field-rect"
-            :width="fieldWidth"
-            :height="fieldHeight"
-          />
+    <svg class="grapher-svg"><g>
```

Comment:
Can you indent this `<g>` element
```suggestion
    <svg class="grapher-svg">
      <g>
```